### ISSUE TITLE
TS issues

### ISSUE THREAD

**Issue by rwwagner90** [orbitjs/ember-orbit#314]
I just started trying to use ember-cli-typescript, and it's complaining about missing types from the private folder of this addon. Any suggestions or workarounds?

```
node_modules/ember-orbit/index.d.ts:1:47 - error TS2307: Cannot find module './-private/factories/coordinator-factory' or its corresponding type declarations.

1 export { default as CoordinatorFactory } from './-private/factories/coordinator-factory';
                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/ember-orbit/index.d.ts:2:42 - error TS2307: Cannot find module './-private/factories/key-map-factory' or its corresponding type declarations.

2 export { default as KeyMapFactory } from './-private/factories/key-map-factory';
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/ember-orbit/index.d.ts:3:42 - error TS2307: Cannot find module './-private/factories/schema-factory' or its corresponding type declarations.

3 export { default as SchemaFactory } from './-private/factories/schema-factory';
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/ember-orbit/index.d.ts:4:48 - error TS2307: Cannot find module './-private/factories/memory-source-factory' or its corresponding type declarations.

4 export { default as MemorySourceFactory } from './-private/factories/memory-source-factory';
                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/ember-orbit/index.d.ts:5:41 - error TS2307: Cannot find module './-private/factories/memory-source-factory' or its corresponding type declarations.

5 export { default as StoreFactory } from './-private/factories/memory-source-factory';
```

**Comment by dgeb** [comment]
I believe these particular issues have been resolved in v0.17.0-beta.7.

**Comment by rwwagner90** [comment]
I can confirm that most issues are resolved. Thanks for fixing this up @dgeb!
